vendor/blockchain_contracts/build.rs

```diff
@@ -10,9 +10,9 @@ use std::{
 };
 
 fn main() -> std::io::Result<()> {
-    compile("./src/ethereum/rfc003/templates/ether_contract.asm")?;
-    compile("./src/ethereum/rfc003/templates/deploy_header.asm")?;
-    compile("./src/ethereum/rfc003/templates/erc20_contract.asm")?;
+    compile("./src/bin/calculate_offsets/ethereum/rfc003/templates/ether_contract.asm")?;
```

Comment:
Because this code is only to be used by the `print_offset` bin.
The aim of this crate is also to be imported as a library.

By putting all the `print_offset` code in the `/bin/` folder we ensure that the library consumer does not use this `print_offset` code.